### ISSUE TITLE
Proposal: LinuxKit command to push vmware.vmdk to remote ESXi datastore 

### ISSUE THREAD

**Issue by ajeetraina** [linuxkit/linuxkit#1751]
**Description**

Currently Linuxkit run command provides only the following options:

```
[root@ajeetraina ~]# linuxkit run vmware
Please specify the prefix to the image to boot
USAGE: linuxkit run vmware [options] prefix

'prefix' specifies the path to the VM image.

Options:
  -cpus int
        Number of CPUs (default 1)
  -disk string
        Path to disk image to use
  -mem int
        Amount of memory in MB (default 1024)
```
There is no option available to push the vmdk file to remote ESXi so as to boot up the VM. 
Currently we have this feature available only for Google Cloud Engine.

@thebsdbox It would be great to have option like:

`$linuxkit push vmware --ESXi address <hostname>  -datastore datastore1  vmware.vmdk`

to push it to remote ESXi datastore.

**Comment by justincormack** [comment]
Sounds good.

**Comment by thebsdbox** [comment]
Plan is to put ovftool into a container allowing us to:
- build Open Virt format VMs
- use the ovftool functionality to push to ESXi or perhaps vCloud AIR.

**Comment by thebsdbox** [comment]
This is partially there with #1882

**Comment by thebsdbox** [comment]
Think this can closed due to #1929 ?

**Comment by justincormack** [comment]
sure
